packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -298,6 +298,239 @@ curl -v -X GET \
 HTTP 200:
 Array of [Application objects](/docs/reference/api/apps/#application-object)
 
+## Policy Simulation operations
+<ApiLifecycle access="ie" />
+> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+### Access Simulation
+Access simulation API is an admin API that evaluates policy and policy rules based on the existing policy rule configuration. The evaluation result simulates what the real world authentication flow is and what policy rules have been applied or matched to the authentication flow.
+
+<ApiOperation method="post" url="/api/v1/policies/simulate" />
+
+#### Request
+The section below explains the request parameters and the properties inside the request body.
+
+##### Request parameters
+| Parameter  | Type   | Description                                       |
+| ---------- | ------ | ------------------------------------------------- |
+| `expand` | String | (Optional) `expand=EVALUATED` to include list of evaluated but not matched policy and policy rules `expand=RULE` include detail about why a rule condition (not)matched|
+
+##### Request Body
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| `policyTypes` | Array| (Optional) Supported PolicyTypes for simulation: OKTA_SIGN_ON, MFA_ENROLL, PROFILE_ENROLLMENT, ACCESS_POLICY. Default `null`, return all types|
+| `appInstance` | String | (Required) The appInstance ID for this simulation |
+| `policyContext.user.id` | String | (Required) The userId for this simulation. Only userId or groupIds allowed, not both  |
```

Comment:
would it be better to expand `userId` to user ID? Similar comment to `groupIds`?